### ISSUE TITLE
Need help with automated language pack installation in code-server docker wrapper

### ISSUE THREAD

**Issue by olegchir** [coder/code-server#7141]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### OS/Web Information

- Web Browser: Chrome or any
- Local OS: Windows, Mac, Linux (latest)
- Remote OS: Host - Fedora 41, Docker container - Ubuntu Bookworm (latest)
- Remote Architecture: x86_64
 
- code-server version:
https://hub.docker.com/layers/codercom/code-server/4.96.2-bookworm/images/sha256-542eaf597605f6d6f280574c18d55bc66f19b88c911883c13ca6dcd53038cb54

### Steps to Reproduce

We've developed an in-house wrapper for code-server that includes a feature allowing developers to set their IDE language through their user profile, rather than having to change it repeatedly in the IDE itself. It's a quite important thing for an international team. We implement this along with other customizations through Docker images built for each developer.

Currently, I'm trying to install language packs using this command in the Dockerfile:

`code-server --install-extension ms-ceintl.vscode-language-pack-es`

And setting the locale configuration in:

`/home/coder/.local/share/code-server/User/argv.json`

With the contents:

```
{
  "locale": "es"
}
```

However, I'm encountering several issues:

1. The IDE still defaults to English when launched
2. When attempting to reinstall the plugin via console, it shows as already installed
3. This occurs whether installing from the marketplace ID or a local file
4. In Command Palette -> Configure Display Language, only English appears in the list of downloaded languages
5. After the marketplace API call completes, the Spanish language pack shows as **NOT DOWNLOADED*
6. Interestingly, when manually navigating to the marketplace page and clicking "install," something gets installed that makes the language pack work

The key question is: What is this additional component that makes it work, and how can I install it programmatically? While I could eventually debug this myself, an immediate solution would be extremely helpful.

Has anyone encountered this issue or knows what might be missing from the automated installation process?

### Expected

The language pack should be available right from the start.

### Actual

English language only.

### Logs

```shell

```

### Does this bug reproduce in native VS Code?

This cannot be tested in native VS Code

### Does this bug reproduce in GitHub Codespaces?

This cannot be tested in GitHub Codespaces

### Are you accessing code-server over a secure context?

- [x] I am using a secure context.

**Comment by liruohan399** [comment]
meet this issue too，this is my workaround

install the language extension and change language from UI，this step will generate a json file called “languagepacks.json”，copy this json file，and exe the following cmds in a new env

> code-server --install-extension ms-ceintl.vscode-language-pack-es
> cp argv.json /home/coder/.local/share/code-server/User/
> cp languagepacks.json /home/coder/.local/share/code-server/

**Comment by code-asher** [comment]
Yeah the problem is that the required language files are not generated when installing from the command line (only happens from the UI).  I think this is a bug with VS Code itself though.
